src/PostgREST/Config.hs

```diff
@@ -219,7 +221,7 @@ readAppConfig dbSettings optPath prevDbUri roleSettings roleIsolationLvl = do
     decodeLoadFiles :: AppConfig -> IO AppConfig
     decodeLoadFiles parsedConfig =
       decodeJWKS <$>
-        (decodeSecret =<< readSecretFile =<< readDbUriFile prevDbUri parsedConfig)
+        (decodeSecret =<< readSecretFile =<< addPgrstVerToDbUri =<< readDbUriFile prevDbUri parsedConfig)
```

Comment:
Maybe not the place to add the variable, a `--dump-config` shows that the parameter is included twice. IO-tests also fail because the `fallback_application_name` is now present in the querystring.